src/types/github.ts

```diff
@@ -102,7 +104,7 @@ export interface Owner {
 
 export interface Subject {
   title: string;
-  url: string;
-  latest_comment_url: string;
+  url?: string;
```

Comment:
The `url` is not yet returned during a `ci_activity` notification response, which is why i changed this - i hope i can get GitHub to fix that up soon but in the meantime let me know if you'd prefer i not change this type and i'll remove it.

Reply:
I think that's absolutely fine - I am hoping that the GitHub GraphQL API will support notifications at some point so these types will go away.